content/en/news/releases/1.5.x/announcing-1.5/change-notes/index.md

```diff
@@ -4,4 +4,93 @@ description: Istio 1.5 release notes.
+- **Added** Authorization Policy supports deny action and exclusion semantics.
+- **Added** Auto mutual TLS is in Beta. The feature is enabled by default.
+- **Added** SDS is in Stable. SDS is enabled by default for identity provisioning.
+- **Improved** Node agent is merged with Pilot agent. This removes the requirement of pod security policy and improves security posture.
+- **Improved** Merge Citadel certificate provisioning functionality into Pilot.
+- **Improved** Support Kubernetes [`first-party-jwt`](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#service-account-tokens) as a fallback token for CSR authentication in clusters that [`third-party-jwt`](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#service-account-token-volume-projection) is not supported.
+- **Improved** Provision key and cert to Prometheus through Istio Agent.
+
+## Telemetry
+
+- **Added** TCP protocol support for v2 telemetry.
+- **Added** gRPC response status code support in metrics/logs.
+- **Added** support for Istio Canonical Service.
+- **Improved** stability of v2 telemetry pipeline.
+- **Added** alpha-level support for configurability in v2 telemetry.
+- **Added** support for populating AWS platform metadata in Envoy node metadata.
+- **Improved** Updated the Stackdriver adapter for Mixer to support configurable flush intervals for tracing data.
+- **Added** support for a headless collector service to the Jaeger addon.
+- **Fixed** `kubernetesenv` adapter to provide proper support for pods that contain a dot in their name.
+- **Improved** the Fluentd adapter for Mixer to provide millisecond-resolution in exported timestamps.
+
+
+## Configuration management
+
+## Operator
+
+**Replaced** the `IstioControlPlane` API with the `IstioOperator` API.
+**Added** `istioctl operator init` and `istioctl operator remove` commands istio/operator#643.
+**Improved** reconciliation speed with caching #20344.
+
+## `area/perf and scalability`
+
+- **Improved** Skip unused services while generate clusters for the gateways #20124.
+- **Improved** Skip calling `updateEDS` for headless service #18952.
+- disable `SNI-DNAT` at ingress gateway by default #18431.
+- The declaration of err overrides #19473.
+- create slice with capacity, when capacity is known #18504.
+
+## `area/test and release` - Triage done
+
+- Build a docker image for `istioctl` #19079.
+
+## `istioctl`
+- **Added** mTLS analyzer - #18350
+- **Added** JwtAnalyzer - #20812
+- **Added** ServiceAssociationAnalyzer #19383
+- **Added** SercretAnalyaer #19583
+- **Added** sidecar ImageAnalyzer  #20929
+- **Added** PortNameAnalyzer #19375
+- **Added** Policy DeprecatedAnalyzer #20919
+- **Updated** more validation rules for `RequestAuthentication` #19369.
+
+- **Graduated** `istioctl analyze` out of experimental  #19488.
+
+- **Added** a new flag `-A|--all-namespaces` to `istioctl analyze` to analyze the entire cluster #19209.
```

Comment:
Should this have commas or perhaps enclose the argument in parenthesis? 

**Added** a new flag, `-A|--all-namespaces`, to `istioctl analyze` to analyze the entire cluster #19209.